docs/circuits.ipynb

```diff
@@ -590,7 +587,7 @@
     "id": "e31b4ff57134"
    },
    "source": [
-    "### Nesting circuits with CircuitOperation\n",
+    "## Nesting circuits with CircuitOperation\n",
```

Comment:
It would. I thought it would be useful to have not everything under a single heading, and this seemed like the best place to do that. However, it should still technically be under the constructing heading. Changed back.